vscode/src/copilot/azqTools.ts

```diff
@@ -433,7 +431,7 @@ export async function submitToTarget(
 
   let qir = "";
   try {
-    qir = await getQirForVisibleSource(supportsAdaptive(target.id));
+    qir = "";
   } catch (e: any) {
```

Comment:
I think you forgot to put this back in.

Reply:
This was indeed a mistake. Let me put it back in